Update contributing-coding.md (#9130)
fishtown-analytics/docs.getdbt.com: website/docs/community/contributing/contributing-coding.md

There are three primary ways to contribute to the dbt projects. We’ll use the <Constant name="fusion_engine" /> as an example, as the "front door" to the dbt ecosystem and a great place to start for newcomers:

- [Open an issue](https://github.com/dbt-labs/dbt-fusion/issues/new/choose) to suggest an improvement or give feedback.

> - Comment / engage on existing [issues](https://github.com/dbt-labs/dbt-fusion/issues) or [discussions](https://github.com/dbt-labs/dbt-fusion/discussions). This could be upvoting issues that would be helpful for your organization, commenting to add nuance to a feature request or sharing how a feature would impact your dbt usage.

Reviewer: ```suggestion
- Comment on or engage with existing [issues](https://github.com/dbt-labs/dbt-fusion/issues) or [discussions](https://github.com/dbt-labs/dbt-fusion/discussions). For example, upvote issues that would help your organization, comment to add nuance to a feature request, or share how a feature would impact your use of dbt.
```
